src/com/esotericsoftware/yamlbeans/Beans.java

```diff
@@ -38,253 +38,370 @@
 
 import com.esotericsoftware.yamlbeans.YamlConfig.ConstructorParameters;
 
-/** Utility for dealing with beans and public fields.
- * @author <a href="mailto:misc@n4te.com">Nathan Sweet</a> */
+/**
+ * Utility for dealing with beans and public fields.
+ *
+ * @author <a href="mailto:misc@n4te.com">Nathan Sweet</a>
+ */
 class Beans {
-	private Beans () {
+
+	static private final String GETTER = "get";
+	static private final String GETTER_BOOLEAN = "is";
+	static private final String SETTER = "set";
+
+	private Beans() {
 	}
 
-	static public boolean isScalar (Class c) {
+	static public boolean isScalar(Class c) {
 		return c.isPrimitive() || c == String.class || c == Integer.class || c == Boolean.class || c == Float.class
-			|| c == Long.class || c == Double.class || c == Short.class || c == Byte.class || c == Character.class;
+				|| c == Long.class || c == Double.class || c == Short.class || c == Byte.class || c == Character.class;
 	}
 
-	static public DeferredConstruction getDeferredConstruction (Class type, YamlConfig config) {
+	static public DeferredConstruction getDeferredConstruction(Class type, YamlConfig config) {
+		DeferredConstruction deferredConstruction = null;
+
 		ConstructorParameters parameters = config.readConfig.constructorParameters.get(type);
-		if (parameters != null) return new DeferredConstruction(parameters.constructor, parameters.parameterNames);
+		if (parameters != null) {
+			deferredConstruction = new DeferredConstruction(parameters.constructor, parameters.parameterNames);
+		}
 		try {
 			Class constructorProperties = Class.forName("java.beans.ConstructorProperties");
+
 			for (Constructor typeConstructor : type.getConstructors()) {
 				Annotation annotation = typeConstructor.getAnnotation(constructorProperties);
-				if (annotation == null) continue;
-				String[] parameterNames = (String[])constructorProperties.getMethod("value").invoke(annotation, (Object[])null);
-				return new DeferredConstruction(typeConstructor, parameterNames);
+				if (annotation == null) {

```

Comment:
There is no reason to add curly braces. Please follow the style the project is already using.